x-pack/plugins/security_solution/public/app/home/global_header/index.tsx

```diff
@@ -85,6 +87,11 @@ export const GlobalHeader = React.memo(
                 data-test-subj="add-data"
                 href={href}
                 iconType="indexOpen"
+                onClick={() => {
+                  if (isTourShown) {
```

Comment:
can you please make this function a `useCallback`?

Reply:
sure @stephmilovic, do you mean something like this? 
```
const closeTour = useCallback(...);
...
onClick={closeTour}
...
```